### DISCUSSION TITLE
`TrouSerS ERROR: TCS GetCapability failed with result = 0x1e`

### DISCUSSION THREAD

**Discussion by J-Kappes** [szaimen/Nextcloud-NAS-Guide#123] [Q&A]
Systemd keeps spitting this error at me when I try to install trouSers/tpm-tools. I've skipped the encryption and TPM but the nextcloud installer script also fails due to this.
```
ls -l /dev/tpm*
crw-rw---- 1 tss root  10,   224 Feb 18 12:48 /dev/tpm0
crw-rw---- 1 tss tss  253, 65536 Feb 18 12:48 /dev/tpmrm0
```

**Comment by szaimen** [comment]
> trouSers/tpm-tools

is this something you are installing manually?

**Reply by J-Kappes** [reply]
Yes and no: this error comes up while running the Nextcloud installer causing it to fail.

But initially, I encountered it while trying to install tpm-tools to check for tpm2-support.

**Reply by szaimen** [reply]
I fear I cannot reproduce this :/ 
maybe something is wrong with your OS installation?

**Reply by J-Kappes** [reply]
Perhaps, or a problem with my hardware. I guess I'll try starting over some time to make sure it wasn't the installation.

**Comment by J-Kappes** [comment]
After reinstalling Ubuntu, the error didn't reoccur. Things I did differently as far as I can remember are
1. Not updating the installer at the beginning
2. Choosing to encrypt the ubuntu-vg volume
